Mathlib/SetTheory/Cardinal/Aleph.lean

```diff
@@ -560,6 +564,18 @@ theorem preBeth_omega : preBeth ω = ℵ₀ := by
 theorem preBeth_pos {o : Ordinal} : 0 < preBeth o ↔ 0 < o := by
   simpa using preBeth_lt_preBeth (o₁ := 0)
 
+@[simp]
+theorem preBeth_eq_zero {o : Ordinal} : preBeth o = 0 ↔ o = 0 := by
+  simpa using preBeth_inj (o₂ := 0)
+
+theorem isStrongLimit_preBeth {o : Ordinal} (H : IsSuccLimit o) : IsStrongLimit (preBeth o) := by
```

Comment:
Is this an iff?

Reply:
It is! I'll update the PR to prove this stronger version.